ariadne/cardano/src/Ariadne/System/FileMode.hs

```diff
@@ -0,0 +1,39 @@
+module Ariadne.System.FileMode
+       ( getAccessMode
+       , setMode
+       , ensureModeIs
+       , mode600
```

Comment:
Are you sure that all of these exports are needed? You may run `weeder` on the codebase to see which of them are never used outside of this module.

Reply:
removed the ones not used. The code snippet here hasn't updated to reflect that though.